src/libpgagroal/management.c

```diff
@@ -2195,24 +2226,19 @@ pgagroal_management_read_config_get(int socket, char* config_key, char* expected
 
    if (output_format == COMMAND_OUTPUT_FORMAT_JSON)
    {
-      pgagroal_json_print_and_free_json_object(json);
+      pgagroal_print_and_free_object(json,FORMAT_JSON);
       json = NULL;
       goto end;
    }
-
-   // if here, print out in text format
-   cJSON* output = pgagroal_json_extract_command_output_object(json);
-   cJSON* value = cJSON_GetObjectItemCaseSensitive(output, "value");
-   cJSON* key = cJSON_GetObjectItemCaseSensitive(output, "key");
-   if (verbose)
+   else if(output_format == COMMAND_OUTPUT_FORMAT_TEXT)
    {
-      printf("%s = %s\n", key->valuestring, value->valuestring);
+      pgagroal_json_print(json,FORMAT_TEXT);
```

Comment:
This is much too information according to me.
If `verbose` is specified, the previous version of the application was printing the result and the exit status, e.g.:
```
% pgagroal-cli conf get max_connections --verbose
max_connections = 10
pgagroal-cli: Success (0)
```

Now it is printing out the whole json, that is somehow confusing and difficult to parse:

```
% pgagroal-cli conf get max_connections --verbose
application: 
  major: 2
  minor: 0
  name: pgagroal
  patch: 0
  version: 2.0.0
command: 
  error: 0
  exit-status: 0
  name: conf get
  output: 
    key: max_connections
    value: 10
  status: OK
pgagroal-cli: Success (0)

```

Reply:
I think it about getting the `conf get | set` ported to pgmoneta, and then we can work from there

@Jubilee101 Cc:

Reply:
https://github.com/pgmoneta/pgmoneta/issues/401

Reply:
True I actually thought that by ``verbose`` it meant literally ``verbose`` to print out everything out but now I get it
The second commit is actually more of a trial-and-error tbh but ill rewrite the logic of text format again starting the first commit as a fresh one now that I understand most of the logic

Reply:
@MohanadKh03 Maybe we should start with pgmoneta's 401 to figure out the best way to extract a single value, or set it...

Because the changes will likely be part of pgagroal's 466 anyway

Reply:
> True I actually thought that by `verbose` it meant literally `verbose` to print out everything out but now I get it The second commit is actually more of a trial-and-error tbh but ill rewrite the logic of text format again starting the first commit as a fresh one now that I understand most of the logic Though in `status details` for example isnt it normal to have all the details so the whole object should be printed out right ?

Well, `details` is a very rich command, and it is text printed in a kind of tabular data. We could simplify the output by simply printing out the json object, but I would not change the current text format too much in order to not "scary" users that could have been scripting on the text output.

That's also why `--verbose` should not be the implementation of a `dump` command, according to me.

Reply:
@jesperpedersen After reading pgmoneta's way imo it's much easier to read and maintain tbh **compared to the current pgagroal's way**

So do you suggest holding this PR off until 401 in pgmoneta is done and then we can standardize the IO,Management in both pgmoneta and pgagroal ?

Reply:
@MohanadKh03 Yes, the management part should be unified across pgagroal, pgmoneta and pgexporter.

@decarv is taking care of the I/O part